tests/components/freebox/common.py

```diff
@@ -0,0 +1,27 @@
+"""Common methods used across tests for Freebox."""
+from unittest.mock import patch
+
+from homeassistant.components.freebox.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .const import MOCK_HOST, MOCK_PORT
+
+from tests.common import MockConfigEntry
+
+
+async def setup_platform(hass: HomeAssistant, platform: str) -> MockConfigEntry:
+    """Set up the Freebox platform."""
+    mock_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={CONF_HOST: MOCK_HOST, CONF_PORT: MOCK_PORT},
+        unique_id=MOCK_HOST,
+    )
+    mock_entry.add_to_hass(hass)
+
+    with patch("homeassistant.components.freebox.PLATFORMS", [platform]):
```

Comment:
Same comment as previous PR

Reply:
Same comment then 😆

Emmm, maybe, but should we change all integrations using this method then ?
![image](https://github.com/home-assistant/core/assets/14821482/3ce7e034-0e5f-406e-8dda-562b0ade9185)

Not an expert, but maybe it makes test faster, personally only took exemple.

Reply:
Closed because https://github.com/home-assistant/core/pull/99757#discussion_r1317655160